### PR TITLE
Fixes #24749 - Add org id on cvv show api call.

### PR DESCRIPTION
**Pull request by chris1984**
Part 1 of issue, 2nd part will be a PR in hammer I will open up.

### CODE REVIEW

**Comment by chris1984** [comment]
@beav can I get a review please?

**Comment by chris1984** [comment]
@beav updated to remove the required and just add the param.

**Comment by jlsherrill** [comment]
Why exactly is org_id being added to this api?

File: app/controllers/katello/api/v2/content_view_versions_controller.rb [hunk 1]
```diff
@@ -45,6 +45,7 @@ def index_relation
 
     api :GET, "/content_view_versions/:id", N_("Show content view version")
     param :id, :number, :desc => N_("Content view version identifier"), :required => true
+    param :organization_id, :number, :desc => N_("Organization identifier"), :required => false
```
Comment (akofink) [inline app/controllers/katello/api/v2/content_view_versions_controller.rb]: `required: false` is the default

**Comment by chris1984** [comment]
Ack closing based on discussion.

**Pull request closed without merging**